axelrod/tests/strategies/test_lookerup.py

```diff
@@ -28,6 +27,31 @@ class TestLookerUp(TestPlayer):
     expected_class_classifier = copy.copy(expected_classifier)
     expected_class_classifier['memory_depth'] = float('inf')
 
+    def test_create_lookup_table_keys(self):
+        expected = [
+            ('C', 'C', 'C'), ('C', 'C', 'D'), ('C', 'D', 'C'), ('C', 'D', 'D'),
```

Comment:
I went with what was already there in the existing test, but yes, you're right. I'll fix the whole lot.